src/main/java/edu/oakland/production/database/DatabaseCommInterfaceClass.java

```diff
@@ -0,0 +1,46 @@
+package edu.oakland.production.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabaseCommManager;
+
+public class DatabaseCommInterfaceClass {
```

Comment:
This should be updated to comply with the naming conventions that were laid out in the UML Class Diagrams. You also need to make sure that you implement the `DatabaseCommInterface` Java interface.
```suggestion
public class DatabaseCommInterfaceImplementation implements DatabaseCommInterface {
```